src/components/Steps/index.tsx

```diff
@@ -0,0 +1,40 @@
+import type { Step } from '@/lib/types';
+
+type StepsProps = {
+  steps: Step[];
+};
+
+const Steps: React.FC<StepsProps> = ({ steps }) => {
+  return (
+    <section className="text-gray-100 body-font">
+      <div className="px-5 py-32 lg:px-0">
+        <div className="flex flex-wrap -mx-4 -mt-4 -mb-10 sm:-m-4">
+          {steps
+            .sort((a, b) => (a > b ? -1 : 1))
```

Comment:
Does this work? Shouldn't it be like this?
```suggestion
            .sort((a, b) => (a.step - b.step))
```

Reply:
@SebasEC96 i tested and work, both are options for order the array asc, i can commit your option if you like

Reply:
Are we sure that the array is not coming ordered ascendent from de sanity api?

Reply:
the query don't have an order in the steps field

Reply:
the order probably is the position in the array, that can be modified in the sanity

Reply:
Choose the option you like the most and we merge it 😛